docs/docs/behaviors/layers.md

```diff
@@ -57,6 +57,14 @@ Example:
 &lt LOWER SPACE
 ```
 
+:::info
+Functionally, the layer-tap is a ["hold-preferred" flavor](hold-tap.md/#flavors) of [hold-tap](hold-tap.md) with a [`tapping-term-ms`](hold-tap.md/#tapping-term-ms) of 200 that takes in a [`momentary layer`](#momentary-layer) and a [keypress](key-press.md) as its "hold" and "tap" parameters respectively.
+
+For users looking to create a keybind like the layer-tap that depending on how long the key is held, invokes behaviors like [sticky keys](sticky-key.md) or [key toggles](key-toggle.md), see [Hold-Tap](hold-tap.md).
+
+Similarly, for users who want to send a different [keycode](../codes/index.mdx) depending on if the same key is held or tapped, see [Mod-Tap](mod-tap.md).
```

Comment:
How about swapping these paragrpahs to mention mod-tap first, then hold-tap? This would go in the order of specific to more general.